attestation/verifier-risczero/methods/guest/src/main.rs

```diff
@@ -390,6 +421,53 @@ fn main() {
     let signature = Signature::from_scalars(r, s).unwrap();
 
     verifying_key.verify_prehash(&hash, &signature).unwrap();
+}
 
-    println!("Done!");
+#[cfg(test)]
+mod tests {
+    use std::cell::RefCell;
+    use std::rc::Rc;
+
+    use crate::verify;
+
+    // TODO: add more extensive tests
+
+    // NOTE: Seems a bit convoluted, idk if it can be simplified
+    fn create_committer() -> (Rc<RefCell<Vec<u8>>>, impl Fn(&[u8])) {
+        let env = Rc::new(RefCell::new(vec![]));
+
+        let env_clone = env.clone();
+
+        let commit_slice = move |slice: &[u8]| {
+            env_clone.borrow_mut().extend_from_slice(slice);
+        };
+
+        (env, commit_slice)
+    }
+
+    #[test]
+    fn test_aws() {
```

Comment:
we can add comments about how the testcase file and expected bytes  is generated.